cdap-docs/developers-manual/source/building-blocks/datasets/index.rst

```diff
@@ -178,6 +177,11 @@ such as the format and schema, while abstracting from the actual underlying file
 - An extension of FileSets, ``PartitionedFileSets`` allow the associating of meta data (partitioning keys)
   with each file. The file can then be addressed through its meta data, removing the need for programs to
   be aware of actual file paths.
+  
+- A ``TimePartitionedFileSet`` is a further variation that uses a timestamp as the partitioning key.
+  Unlike a :ref:`Timeseries Table <cdap-timeseries-guide>` dataset, where the same schema is
```

Comment:
Not so happy with this. The main difference between time series and time-partitioned is the data model. In a time-partitioned file set, data in each partition is not (necessarily) organized by time, but each partition is assigned a logical time. In a time series, time is the primary means of how data is organized, the data model and the schema that represents the data are optimized for querying and aggregations over time range. Moreover, time partitions are typically written in batch (on large file very N minutes/hours/...), whereas time series are typically written in real-time, one data point at a time. 

However, although what you wrote about schema is true, it is misleading, because typically, all partitions of a partitioned file set have the same schema. The schema may evolve over time, but that is also true for time series.